logstash-core/src/main/java/org/logstash/execution/queue/LegacyMemoryQueueWriter.java

```diff
@@ -26,6 +26,9 @@
 import org.logstash.RubyUtil;
 import org.logstash.ext.JrubyEventExtLibrary;
 
+/**
+ * Unused class
+ * */
```

Comment:
This was introduced in e89501e47e9efe9607cf008dbb43e634abb6e1a3 and has _never_ been used. I'm in favor of deleting.